data/199495/ipq-199495--0009.xml

```diff
@@ -3,93 +3,117 @@
   <teiHeader>
     <fileDesc>
       <titleStmt>
-        <title>Interpellation Question</title>
+        <title>de ekono­miskpolitiska målen vid ett EU-medlemskap</title>
+        <author>Per-Ola Eriksson (c)</author>
       </titleStmt>
       <publicationStmt>
         <authority>SWERIK PROJECT</authority>
       </publicationStmt>
       <sourceDesc>
         <bibl>
-          <title>Interpellation Question</title>
+          <title>1994/95:9 av Per-Ola Eriksson (c) till finansministern om de ekono­miskpolitiska målen vid ett EU-medlemskap</title>
+          <ref type="dokument_url_pdf">https://media.githubusercontent.com/media/swerik-project/interpellation-questions-pdf/main/data/199495/pages/ip_199495_9.pdf</ref>
+          <ref type="dokument_url_pdf">https://media.githubusercontent.com/media/swerik-project/interpellation-questions-pdf/main/data/199495/pages/ip_199495_10.pdf</ref>
         </bibl>
       </sourceDesc>
     </fileDesc>
-    <encodingDesc>
-      <editorialDecl>
-        <correction>
-          <p>No correction of source texts was performed.</p>
-        </correction>
-      </editorialDecl>
-    </encodingDesc>
+    <profileDesc>
+      <particDesc>
+        <listPerson>
+          <person gender="man">
+            <persName>Per-Ola Eriksson (c)</persName>
+            <state ref="unknown" type="partyAffiliation">
+              <desc>c</desc>
+            </state>
+            <idno>i-GyYMjkDFkNwtQ7GJKMBnVE</idno>
+          </person>
+          <person>
+            <persName>finansministern</persName>
```

Comment:
Later on, we want to map this to the actual person

Reply:
that's the plan